paths/figma_attachment_keys/destroy.yaml

```diff
@@ -40,4 +34,16 @@ x-code-samples:
       --id <id> \
       --branch my-feature-branch \
```

Comment:
this should then go to `--data` parameter instead, I guess?

Reply:
Yeah right. That makes me now more hesitant to move them all form query params to the body. Technically this is probably correct but also requires many changes and saw that I've still missed lots of branch query params

Reply:
I've reverted these changes and just focused on required params